<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>website</class>
 <widget class="QMainWindow" name="website">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>website</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/img2.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="kepler30">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kepler-30 : arXiv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kepler29">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kepler-29 : arXiv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rem">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>REM : arXiv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bologna">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Physycom.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>PORTFOLIO PANICHI FEDERICO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>401</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf_k60">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kepler-60: PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf_k30">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kepler-30: PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf_k29">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kepler-29: PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf_rem">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>REM: PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf_k60_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kepler-60: arXiv</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
